fairscale/experimental/wgit/repo.py

```diff
@@ -356,11 +456,3 @@ def _rel_file_path(self, filepath: Path) -> Path:
             pass
         # return the relative part (path not common to cwd)
         return Path(*filepath.parts[i:])
-
-
-class RepoStatus(Enum):
```

Comment:
this is moved to the top because during mypy checking, this class is needed earlier.